### DISCUSSION TITLE
앞으로의 미션에 대한 각자의 생각

### DISCUSSION THREAD

**Discussion by lokba** [woowacourse-study/2022-code-review-study#51] [Youtube] [answered]
요즘 리뷰들을 보면, 
JS 기본기, JS 문법, 클린코드에 관련된 리뷰가 확연히 줄었다고 생각이 드네요.
그만큼, 전체적인 크루분들의 코드가 많이 가독성이 좋아지고 JS를  잘 다룬다고 생각이 듭니다.

제가 8명의 코드를 보면서, 대체적으로 코드의 느낌이 비슷비슷하다였습니다.
네이밍 괜찮고, 함수 분리 잘 되어있고, 가독성도 좋다.

그 중에서, 몇명 크루분들은 각자가 공부해서 새로운걸 적용하는 분들이 있더라고요.

남은 자판기 미션에서, 어떤 생각을 가지고 임할지 고민이네요.

여러분은 어떤 생각을 하고 있나요?

**Comment by wonsss** [comment]
Web Components, custom elements 써보고 싶어요.

**Comment by moonheekim0118** [comment]
전 그냥 단순합니다
미션에서 주어진 문제해결에 집중하고싶어요 그게 새로운 방법이든, 기존에 썼던 방법이든
도구에 매몰되지 않고 목적(문제해결)에 집중하고싶어요 ~

**Reply by wonsss** [reply]
그러면 MVC...는 어떠신가요?

**Reply by moonheekim0118** [reply]
MVC가 문제해결에 적합하다면 ㅇㅋ죠

**Comment by KangYunHo1221** [comment, marked as answer]
호프에 동의합니다. 
연습이니까 
프로젝트에 안어울리는 설계를 연습했을때는 기억에 더 오래 남을 것 같아요. 이건 이런 장단점이 있구나 하구요.
반대로 어울리는 설계를 연습했을때는, 아 이럴때는 이런식으로 구조를 선택하면 유리하구나 하는걸 배울수 있고요.

저도 그냥 문제 해결에 집중해서 구조는 마음가는대로  할 것 같습니다.

오히려 주어진 과제 이외에 추가하고 싶은 기능들을 먼저 생각해볼 것 같아요!

**Comment by kkojae91** [comment]
저는 우선적으로 관심사 분리와 custom event를 사용해보려구요!
